apiary.apib

```diff
@@ -719,12 +719,35 @@ API endpoints that Operation Code's Rails backend makes available to its React f
 
 + Response 400 (application/json)
 
+### List All Mentors [GET]
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "mentor": name
```

Comment:
To get a sense of what is returned from a given endpoint response, you can:

1. `byebug` into the associated test
2. check what the `response.body` looks like

For example, for the index action you could:

1.  in here  https://github.com/OperationCode/operationcode_backend/blob/master/test/controllers/api/v1/mentors_controller_test.rb#L5-L11
2. comment out the `skip`
3. add a `byebug` below line 10
4. call `response.body`


And then you would follow the same general pattern for all the other endpoints you are adding documentation for.

Reply:
@dave5801 - This response is what needs updating.  Right now it is showing up as:

```
{
    "mentor": name
}
```

Same applies to the "services collection" and the "status collection" endpoints.